Automattic/jetpack
# Search: Fix Jetpack Search widget name

#### Changes proposed in this Pull Request:
Fix Jetpack Search widget name to differentiate from core search widget. 

Note:
- WPCOM uses `Search (Powered by Jetpack)` as weidget name.
- Jetpack and Search plugins use `Search (Jetpack)`

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
- Prepare a site with JP Search plugin only, connect site and purchase a subscription
- Navigate to `wp-admin/widgets.php`
- Click Add, search `search`
- Ensure JP Search widgets is show as `Search (Jetpack)`

![image](https://user-images.githubusercontent.com/1425433/171084384-ec710a5a-99ee-4182-80b2-9a7de0fde54f.png)
